### DISCUSSION TITLE
Waste of screen space

### DISCUSSION THREAD

**Discussion by susantabiswas** [community/community#37594] [Other Feature Feedback, Questions, & Ideas]
This provides way lesser information compared to the previous one. Also, there is so much wastage of screen space, I see only a few things and they have huge spacing between them.

**Comment by pawamoy** [comment]
I agree. Probably a question of habit, but I find to new layout hard to scan. Previously I was able to see that this user starred this repo in one go, now I see a repository (??), and I have to look up to see that it was starred/forked/whatever.

**Reply by susantabiswas** [reply]
Also noticed that it doesn't show if anybody has started following you.

**Comment by ossanna16** [comment]
Hi there @susantabiswas and welcome to our community! Thank you for the feedback!
